neo/public/stylesheets/cloudberry.css

```diff
@@ -0,0 +1,77 @@
+/*
+* multi-line comment
```

Comment:
Is this file for instruction purpose only? Or did you use it in the admin console?